src/js/components/Ticker.jsx

```diff
@@ -66,22 +66,16 @@ const Ticker = ({value, amount, rate, tickTime=1000, currencySymbol = '', pretty
 	};
 
 	let disp = pretty? formatNum(dispVal) : dispVal.toString();	
+	disp = currencySymbol + disp;
 
-	// Get the total value in pretty penny form too, for preserving the size
-	let totalVal = pretty ? formatNum(value) : value.toString();
-	
-	// Make sure the display value is no longer than the end size
-	disp = disp.substr(0, totalVal.length);
-
-	// To avoid having the surrounding text jitter, we fix the size.
-	// using an invisible final value to get the sizing right.
-	// Text is aligned by absolute position of span, right:0 = right alignment by default
-	// If centerText is set, width is set to 100 and text-center does the job
-	// When centerText is set, the container div gets some extra horizontal padding to stop text overflow
+	let dispArr = disp.split("");
 	return (
-		<div className="position-relative d-inline-block" style={{padding: "0 " + (centerText ? "0.1rem" : "0")}}>
-			<span className="invisible text-center" style={{width: centerText ? "100%" : "auto"}}>{currencySymbol + totalVal}</span>
-			<span className="position-absolute text-center" style={{right: 0, width: centerText ? "100%" : "auto"}} ref={ref}>{currencySymbol + disp}</span>
+		<div className="position-relative flex-row justify-content-center align-items-center" style={{padding: "0 " + (centerText ? "0.1rem" : "0")}}>
+			{/*<span className="invisible text-center" style={{width: centerText ? "100%" : "auto"}}>{currencySymbol + totalVal}</span>
+			<span className="position-absolute text-center" style={{right: 0, width: centerText ? "100%" : "auto"}} ref={ref}>{currencySymbol + disp}</span>*/}
+			{dispArr.map(digit => <div style={{width:unitWidth || "1rem", textAlign:"center", margin:"unset"}}>
+				{digit}
```

Comment:
map() outputs are missing the `key` prop

Reply:
Fixed